Simple-PHP-IPAM/assets/app.css

```
@@ -95,90 +148,49 @@
+   Used to pair color-only signals (util-bar severity, health-dot status)
+   with text equivalents for WCAG 1.4.1. Two existing consumers in
+   change_password.php were relying on this class without a definition;
+   defining it here fixes that latent bug too. */
+.sr-only{
+  position:absolute;
+  width:1px;
+  height:1px;
+  padding:0;
+  margin:-1px;
+  overflow:hidden;
+  clip:rect(0,0,0,0);
```

Stylelint (17.11.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)